kitchen.docker.yml

```diff
@@ -1,6 +1,7 @@
 ---
 driver:
   name: dokken
+  platform: linux/amd64
```

Comment:
I am not sure we should set this as the default, as this is not the most common architecture.

Reply:
no other platform can currently be supported with dokken, `chef_image: cincproject/cinc` only exists for `amd64` and is there already set an env that is `amd64` specific: `KERNEL_RELEASE=3.10.0-1160.42.2.el7.x86_64`

Reply:
Yes, sorry, of course you are correct, I had mistakenly read `arm64` there. I am okay, with this change.